pom.xml

```diff
@@ -35,6 +35,7 @@
 
     <kie.download.build.version>7.48.0.Final</kie.download.build.version>
     <kogito.download.build.version>2.0.0-SNAPSHOT</kogito.download.build.version>
+    <optaplanner.download.build.version>8.3.0.redhat-2021032300</optaplanner.download.build.version>
```

Comment:
which version should we have here ?

I am puzzled because I see one Final, one SNAPSHOT and one redhat ;)

Reply:
It do not sure what to put here
Though I believe that there is a mistake in kogito.download.build.version
Their maven-dependency-plugin configured for redhat version 
https://github.com/kiegroup/run-tests-with-build/blob/d58bea6d6036ae1397624c160bcd9c8486b9d860/run-tests-with-build-kogito-examples/pom.xml#L41
(See project-sources name of the packages)
Maybe it would be better to use 8.4.1.Final version then. @tsurdilo can I change a pipeline to automatically update that version in this repository as it made in kogito-eng-pipeline?

Reply:
Moved to FINAL version however that is not necessary that parameter should be changed automatically